fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/socialmedia/SocialMediaViewModel.kt

```diff
@@ -109,7 +112,7 @@ constructor(
     val addChirp = AddChirp(text, parentId, timestamp)
 
     return Single.fromCallable { validPoPToken }
-        .doOnSuccess { token: PoPToken ->
+        .doOnSuccess { token: PoPToken? ->
```

Comment:
Why did you change that? Technically the token cannot be null there: if a KeyException error is thrown and then become null, the doOnSuccess won't be triggered anyway (as it won't be a success)

Reply:
.... It helped raising the coverage 😅 (since the tests are having trouble creating valid data, some of them just could not get passed this without the "?"